### PR TITLE
create feat/introduction-card

### PR DESCRIPTION
**Pull request by liemht23**

### CODE REVIEW

File: src/components/introduction-card/IntroductionCard.vue [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            alt="promote-man.png"
+          />
+        </q-card-section>
+      </q-card-section>
+    </q-card>
+  </div> 
+</template>
+
+<style lang="sass" scoped>
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&display=swap')
```
Comment (giahuy08) [inline src/components/introduction-card/IntroductionCard.vue:50]: help me update font global-style.css

File: src/components/introduction-card/IntroductionCard.vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  background: #3BB77E
+.card-bg-orange
+  background-color: rgba(255,245,225,.85)
+.card-bg-green
+  background-color: rgba(210,239,225,.85)
+.intro-card-container
+  background-image: url(../../assets/images/intro-card-bg.png)
+  max-width: 590px
+  width: 100%
+.intro-card
```
Comment (giahuy08) [inline src/components/introduction-card/IntroductionCard.vue:65]: where are curly brackets ? I think you use scss syntax like this:
image

File: src/components/introduction-card/IntroductionCard.vue [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            :src="introductionCard.imgUrl"
+            alt="promote-man.png"
+          />
+        </q-card-section>
+      </q-card-section>
+    </q-card>
+  </div> 
+</template>
+
+<style lang="sass" scoped>
```
Comment (giahuy08) [inline src/components/introduction-card/IntroductionCard.vue:49]: I think we use scss to easier to code,  don't you?

File: src/components/introduction-card/IntroductionCard.vue (hunk 1 above)
Comment (giahuy08) [inline src/components/introduction-card/IntroductionCard.vue:50]: help me update font global-style.css

File: src/components/introduction-card/IntroductionCard.vue (hunk 3 above)
Comment (liemht23) [reply src/components/introduction-card/IntroductionCard.vue:49]: Why dont you ask me at the beginning? But it's okay, let me fix it

File: src/assets/css/global-styles.css [hunk 4]
```diff
@@ -1,4 +1,5 @@
 @import url('https://fonts.googleapis.com/css2?family=DM+Sans:opsz,wght@9..40,100;9..40,200;9..40,300;9..40,400;9..40,500;9..40,600;9..40,700;9..40,800;9..40,900;9..40,1000&display=swap');
```
Comment (giahuy08) [inline src/assets/css/global-styles.css:1]: remove this kkkk

**Pull request merged**